docs/modules/ROOT/partials/getting-started/proc-configuring-registry-ui.adoc

```diff
@@ -0,0 +1,36 @@
+
+[id="configuring-registry-ui"]
+= Configuring {registry} User Interface
+
+You may configure the {registry} User Interface in a number of ways, either to customize its behavior or to properly
+configure it for your deployment environment.
+
+== Configuring {registry} UI for Deployment Environment
```

Comment:
Suggest sentence case for all headings:
Configuring the {registry} web console for your deployment environment